.env

```diff
@@ -0,0 +1,2 @@
+
+GEOAPIFY_API_KEY=75443e8e914340ad836ba591601e3c92
```

Comment:
If you're going to use an env file, then it shouldn't be committed to git. Defeats the purpose of moving the secret if it's still on GitHub. Instead, create an `env.example` file, and push that to git instead. Add the main `.env` to git ignore.

Reply:
noted